CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# [Unreleased](https://github.com/purecloudlabs/genesys-cloud-client-logger/compare/v4.2.2...HEAD)
+# [Unreleased](https://github.com/purecloudlabs/genesys-cloud-client-logger/compare/v4.2.3...HEAD)
+# [v4.2.3](https://github.com/purecloudlabs/genesys-cloud-client-logger/compare/v4.2.2...v4.2.3)
+### Fixed
+* [PCM-2088](https://inindca.atlassian.net/browse/PCM-2088) – Respect the retry-after header returned by a 429 response.
```

Comment:
It looks like the description here isn't accurate. :-)

Reply:
woah. don't know where that came from... thanks